debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/Filters.java

```diff
@@ -22,13 +22,24 @@
     protected static final Set<String> BUILT_IN_DB_NAMES = Collect.unmodifiableSet("local", "admin");
 
     private final Predicate<CollectionId> collectionFilter;
+    private final Predicate<String> databaseFilter;
 
     /**
      * Create an instance of the filters.
      * 
      * @param config the configuration; may not be null
      */
     public Filters(Configuration config) {
+        String dbWhitelist = config.getString(MongoDbConnectorConfig.DATABASE_WHITELIST);
+        String dbBlacklist = config.getString(MongoDbConnectorConfig.DATABASE_BLACKLIST);
+        if (dbWhitelist != null && !dbWhitelist.trim().isEmpty()){
```

Comment:
I am sorry but it seems that space is missing between rightmost parenthesis and the left curl brace following